lib/ActiveOrm.js

```diff
@@ -8,7 +8,28 @@ module.exports = class ActiveOrm {
   }
 
   setAttribute(key, value) {
-    this[key] = value;
+    this[`db_${key}`] = value;
```

Comment:
Ok, so here's the prefix! :)

Why did you make this change?  This is the kind of thing you should note in the pull request description.

I know why: because you want your attributes to have their own namespace.  Without their own namespace, a database field like `length`, `prototype`, etc. would cause big problems.

Reply:
That said, the attributes should be specified somewhere beforehand and at the class level.  For example, something like this:

```
class User extends BaseModel {
}

User.setAttributes(['id', 'first_name', 'last_name', 'email']);
```

Then I might have a `Map` field valued `this.attributes` that is set to whatever is stored in `User.attributes` when a `User` is initialized.  Then `getAttributes` becomes:

```javascript
getAttribute(key) {
  if (!this.attributes.has(key)) {
    throw new UnknownAttributeError(`Unknown attribute '${key}' for model '${this.constructor.name}.'`);
  }

  return this.attributes.get(key);
}
```